### ISSUE TITLE
Bossa 1.9 and 1.8 crash when trying to write

### ISSUE THREAD

**Issue by DTS188** [shumatech/BOSSA#119]
This happens on v1.8 and 1.91. As soon as I hit write, Bossa closes or crashes. I need a fix fast.

**Comment by rabryan** [comment]
How fast and how much you willing to pay for this fix?

- Richard

**Comment by DTS188** [comment]
you're kidding

**Comment by rabryan** [comment]
No, no I’m not.  Though I’m not the creator of this software, I do know that it is free (as in freedom) software which gives you (for free) the ability to use and fix this software yourself.

I am a professional software engineer with 15 years of experience who is able to fix this bug that, as you indicated, is a big issue for you.  I was simply offering to help you.  And my first step in helping you was to ask a straightforward question to learn more about your issue.

- Richard

**Comment by DTS188** [comment]
That wasn't question to learn more about the problem. That was a question that appeared to be a scam for money.
I am not willing to pay anything for this. I have another solution. Send the product back to the manufacturer and have them re-program the device. Something I would rather not do, but that doesn't cost me anything.

**Comment by rabryan** [comment]
Ok.  Glad to hear you have a solution then so the issue can be considered resolved.

- Richard

**Comment by DTS188** [comment]
No, I have a secondary solution. I would still like to know why this happens.
It's not up to you to decide if this is resolved.

**Comment by rabryan** [comment]
True.  But based on the information you have provided no person on this planet can actually know why the crash you are claiming has occurred. 

- Richard

**Comment by DTS188** [comment]
Problem solved. Used a different programmer that works.
